lualib/resty/events/protocol.lua

```diff
@@ -153,6 +172,14 @@ function _Client:connect(addr)
         end
     end -- subsystem == "http"
 
+    WORKER_INFO.id = worker_id()
```

Comment:
LGTM. Just wonder if we can add robustness check on the return value.

Reply:
This line should be `worker_id() or -1`, for privileged worker.

Reply:
Added.